lib/between_meals/knife.rb

```diff
@@ -80,6 +93,20 @@ def cookbook_upload(cookbooks)
       end
     end
 
+    def berks_cookbook_upload(cookbooks)
+      # cookbooks: array
+      # cookbook_paths: array
+      if cookbooks.any?
+        @cookbook_dirs.each do |path|
+          cookbooks.each do |cb|
+            @logger.warn("Running berkshelf on cookbook: #{cb}")
```

Comment:
right, I follow now. I did that because @cookbook_dirs is an array, likely stemming from the fact that its an array in the default knife.rb config. In practice I only have one 'cookbooks dir' that I'm working with, so for my purposes I could just use the first item, but I wasn't sure that would be good behavior if a use does have multiple cookbook dirs. Although this isn't much better behavior I suppose. 

Would it make sense to include the cookbook dir the changed cookbook is coming from in the output of your change detection lib (`changes/cookbook.rb`)? If so then I could take that into account and just walk those dirs. Not sure if thats the best way to handle this. 

Or we could just say this feature only supports a single cookbook dir, and take the first item in the `@cookbook_dirs` array.